spec/wor/batchifier/batchifier_spec.rb

```diff
@@ -124,6 +124,19 @@
           expect(batchified_process).to eq(expected_response)
         end
       end
+
+      context 'when an error occurs while processing' do
```

Comment:
I would add the opposite case too, that it should not rise any exceptions on valid strategies.